publish.sbt

```diff
@@ -35,17 +34,10 @@ ThisBuild / developers := List(
 
 ThisBuild / homepage := Some(url("https://github.com/AbsaOSS/fa-DB"))
 ThisBuild / description := "DB data access via DB functions"
+
+// licenceHeader check:
+ThisBuild / organizationName := "ABSA Group Limited"
 ThisBuild / startYear := Some(2022)
 ThisBuild / licenses += "Apache-2.0" -> url("https://www.apache.org/licenses/LICENSE-2.0.txt")
 
-ThisBuild / pomIncludeRepository := { _ => false }
-ThisBuild / publishTo := {
-  val nexus = "https://oss.sonatype.org/"
-  if (isSnapshot.value) {
-    Some("snapshots" at s"${nexus}content/repositories/snapshots")
-  } else {
-    Some("releases" at s"${nexus}service/local/staging/deploy/maven2")
-  }
-}
-ThisBuild / publishMavenStyle := true
-
+ThisBuild / sonatypeProfileName := "za.co.absa"
```

Comment:
if not specified, `ThisBuild / organization` will be used (ref: https://github.com/xerial/sbt-sonatype), thus it can be removed & will be even consistent with our other libraries, like https://github.com/AbsaOSS/spark-data-standardization/blob/master/publish.sbt